src/Engine/labyrinthFactory.h

```diff
@@ -0,0 +1,25 @@
+//
+// Created by nikit on 15/04/2018.
+//
+
+#ifndef ALGODATII_LABYRINTHFACTORY_H
+#define ALGODATII_LABYRINTHFACTORY_H
+
+typedef struct labyrinth {
+    const int firstDimensionSize;
+    const int secondDimensionSize;
+    const int freeField = 0;
+    const int filledField = 1;
```

Comment:
Meiner Meinung nach sollte das in ein Enum. enum class Field { freeField, filledField }